encoder/vaapiencoder_base.h

```diff
@@ -127,6 +128,11 @@ class VaapiEncoderBase : public IVideoEncoder {
     uint32_t frameRateNum() const {
         return m_videoParamCommon.frameRate.frameRateNum;
     }
+
+    uint32_t fps() const {
+        return m_videoParamCommon.frameRate.frameRateNum / m_videoParamCommon.frameRate.frameRateDenom;
```

Comment:
Is any possible m_videoParamCommon.frameRate.frameRateDenom == 0?
It will crash the application